src/test/resources/org/prebid/server/it/amp/test-appnexus-bid-request.json

```diff
@@ -81,7 +81,8 @@
           ]
         },
         "includewinners": true,
-        "includebidderkeys": true
+        "includebidderkeys": true,
+        "includeformat" : false
```

Comment:
Do we want to add defaulting value in bidder request if incoming `includeformat` flag is not defined?